packages/flutter_tools/test/analyze_continuously_test.dart

```diff
@@ -29,7 +29,7 @@ void main() {
     return server?.dispose();
   });
 
-  group('watch', () {
+  group('analyze --watch', () {
```

Comment:
I'd like to keep them separate because the implementations are separate but I just realized I should rename this to analyze_watch_test.dart